otx/algorithms/detection/adapters/mmdet/hooks/det_class_probability_map_hook.py

```diff
@@ -121,3 +124,127 @@ def forward_single(x, cls_convs, conv_cls):
                     "YOLOXHead, ATSSHead, SSDHead, VFNetHead."
                 )
         return cls_scores
+
+
+class MaskRCNNRecordingForwardHook(BaseRecordingForwardHook):
+    """Saliency map hook for Mask R-CNN model. Only for torch model, does not support OV IR model.
```

Comment:
If you don't mind, can you substitute OV with OpenVINO IR model?

Reply:
Sure